app/views/govuk_publishing_components/components/_label.html.erb

```diff
@@ -18,10 +19,10 @@
 
 <% if is_page_heading %>
   <%= tag.h1 text, class: "govuk-label-wrapper" do %>
-    <%= tag.label text, for: html_for, class: css_classes %>
+    <%= tag.label text, id: id, for: html_for, class: css_classes %>
   <% end %>
 <% else %>
-  <%= tag.label text, for: html_for, class: css_classes %>
+  <%= tag.label text, id: id, for: html_for, class: css_classes %>
```

Comment:
This all looks good, I'm just curious for a little more detail on the reason for this change. What element needs to reference a label with `aria-describedby`?

Reply:
Thanks @andysellick! This is related to improvements to the Govspeak Preview component in Whitehall - I forgot to attach the trello ticket so I've added that to the PR comment - and related to the [PR to update the button component](https://github.com/alphagov/govuk_publishing_components/pull/3088). The button will use `aria-describedby` to reference the label ID.